src/pretix/control/templates/pretixcontrol/orders/cancel.html

```diff
@@ -35,8 +35,8 @@ <h1>{% trans "Cancel event" %}</h1>
         {% endif %}
         <fieldset>
             <legend>{% trans "Refund options" %}</legend>
-            {% bootstrap_field form.auto_refund layout="control" %}
             {% bootstrap_field form.manual_refund layout="control" %}
```

Comment:
I've swapped the two, as I felt it would be more naturally how we think about the cancellation process. Also the `manual_refund` option has the extensive help text, so I'd prefer people to read that first.

I'll revert the two options, as it does not matter too much.